guacamole-ext/src/main/java/org/apache/guacamole/properties/GuacamoleProperty.java

```diff
@@ -49,5 +51,25 @@
      *                            provided value.
      */
     public Type parseValue(String value) throws GuacamoleException;
+    
+    /**
+     * Parses the given string value into a sorted Collection of values of the
```

Comment:
What do you mean by "sorted" here? Is the order of the values not determined by the property implementation?